src/python.gen.spec.ts

```diff
@@ -248,6 +248,25 @@ def run_url_encoded_query(
     })
   })
 
+  describe('body params type assertions' , () => {
```

Comment:
I'd imagine you need to update the [assert response is model add_group_group](https://github.com/looker-open-source/sdk-codegen/blob/7925bdb363fd060dfa8b1e05cf6d92007a149c25/src/python.gen.spec.ts#L271) test too?

Reply:
That test focusses on the http call and hence does not include the statement where we assert the type for body parameters. I am going to move my tests under the `method body` describe block though.